MAUI/DataForm/Editors.md

```diff
@@ -0,0 +1,1069 @@
+
+{% tabs %}
+{% highlight MainPage.xaml.cs %}
+
+    [Display(Name = "Is Billable")]
+    public bool IsBillable { get; set; } = true;
+
+    [Display(Name = "Registered Member")]
+    public bool RegisteredMember { get; set; } = false;
+
+    [Display( Name = "Is Refundable")]
+    public bool? IsRefundable { get; set; } = null;
+
+{% endhighlight %}
+{% endtabs %}
+		
+## Switch Editor
+
+In switch editor, [Switch]`Link` is loaded, and DataForm `Switch` editor supports bool data type property.
+
+To add `Switch` editor in DataForm, register the editor as `Switch` for the required property using the [RegisterEditor]`Link` method.
+
+{% tabs %}
+{% highlight MainPage.xaml.cs %}
+
+       dataForm.RegisterEditor("CellularData", DataFormEditorType.Switch);
+       dataForm.RegisterEditor("AirplaneMode", DataFormEditorType.Switch);
+
+        [Display(Name ="Cellular Data")]
+        public bool CellularData { get; set; } = true;
+
+        [Display(Name = "Airplane Mode")]
+        public bool AirplaneMode { get; set; }
+ 
+
+{% endhighlight %}
+{% endtabs %}
+
+`Image`
+
+## Combo Box editor
+
+In the Combo Box editor, the [SfComboBox]`Link` will be loaded.
+
+### Customizing ItemsSource of SfComboBox
+
+By default, the `ItemsSource` for SfComboBox is auto-generated for enum types and collection type properties. For other types, you can set the `ItemsSource` by using the [IDataFormSourceProvider]`Link`.
+
+#### Using IDataFormSourceProvider
+
+{% tabs %}
+{% highlight MainPage.xaml.cs %}
+
+    public string ItemName { get; set; }
+    
+    public class SourceProviderExt : IDataFormSourceProvider
+    {
+        public override IList GetSource(string sourceName)
+        {
+            var list = new List<string>();
+            if (sourceName == "ItemName")
+            {
+                list.Add("Item1");
+                list.Add("Item2");
+                list.Add("Item3");
+            }
+            return list;
+        }
+    }
+    dataForm.ItemsSourceProvider = new SourceProviderExt();
+    dataForm.RegisterEditor("ItemName", DataFormEditorType.ComboBox);
+
+{% endhighlight %}
+{% endtabs %}
+
+#### Using ItemsSource property
+
+You can also set the ItemsSource for combo box editor by using the [ItemsSource]`Link` property in the `DataFormComboBoxItem`.
+
+{% tabs %}
+{% highlight MainPage.xaml.cs %}
+
+    dataForm.GenerateDataFormItem += DataForm_GenerateDataFormItem;
+
+    private void DataForm_GenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem != null && e.DataFormItem.FieldName == "Name")
+        {
+            var list = new List<string>();
+            list.Add("Home");
+            list.Add("Food");
+            list.Add("Utilities");
+            list.Add("Education");
+            (e.DataFormItem as DataFormComboBoxItem).ItemsSource = list;
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+`Image`
+
+### Changing ItemsSource of SfComboBox at run time
```

Comment:
Runtimesupport not provided, so remove this topic

Reply:
runtime content removed